composer.json

```diff
@@ -50,7 +50,8 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Knp\\DoctrineBehaviors\\Tests\\": "tests"
+            "Knp\\DoctrineBehaviors\\Tests\\": "tests",
+            "Knp\\DoctrineBehaviors\\PHPStan\\": "utils/phpstan-behaviors/src"
```

Comment:
How can this work for 3rd party install?

Reply:
For a moment I thought `autoload-dev` would work in 3rd party :sweat_smile: 
Updated.